### DISCUSSION TITLE
How to formulate an exclusion constraint in casadi?

### DISCUSSION THREAD

**Discussion by jargoteg** [casadi/casadi#3384] [Q&A]
Hi,

I'm using casadi as part of my MPC formulation for a robot navigation problem. I have been trying to use logic operators to add a constraint in the form of:
[(lbx1 <= x1 <= ubx1) && (lbx2 <= x2 <= ubx2)]

I tried to find a similar case in the documentation with no luck so far.

Is it possible to add a constraint with logic operators?

Thank you!

**Comment by brunoml5** [comment]
Hi! Can you write it using `if_else`? It is available in the [list of operations](https://web.casadi.org/docs/#general-usage).
